### PR TITLE
the black market thrives in its newfound popularity, harboring more merchants and wares

### PR DESCRIPTION
**Pull request by honkpocket**
## About The Pull Request

The black market has began to flourish now that interests has significantly increases within our sector.

## How This Contributes To The Nova Sector Roleplay Experience

Acceptable gear that can spin a good story for antags and self-antags.

## Proof of Testing
<details>
<summary>Screenshots/Videos</summary>
  
2V5vc2iMDJ

</details>

## Changelog
:cl:
add: Adds new items to the Black Market Uplink: Syndicate clothing, Syndicate duffelbag, Syndicate mask, Military belt, Syndicate spacesuit box, Bulletproof type III armorvest, Anti-slip MOD module, Anti-flash lenses, Syndicate hairtie, Syndicate shotglasses, Dehydrated space-carp, Syndicate smokes, Syndicate stickers, Adaptive cardboard cutouts, AI detector multitool, ablative baseball bat.
/:cl:

### CODE REVIEW

File: modular_nova/master_files/code/modules/cargo/markets/market_items/gadgets.dm [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+	availability_prob = 80
+
+/datum/market_item/misc/c4
+	name = "C4 Satchel Charge"
+	desc = "Sticky charges listed on the black market uplink - surely these won't explode within your own hands..."
+	item = /obj/item/grenade/c4
+	price_min = CARGO_CRATE_VALUE
+	price_max = CARGO_CRATE_VALUE * 2
+	stock_max = 4
+	availability_prob = 80
```
Comment (SomeRandomOwl) [inline modular_nova/master_files/code/modules/cargo/markets/market_items/gadgets.dm:17]:
```suggestion
/datum/market_item/misc/c4
	name = "C4 Satchel Charge"
	desc = "Sticky charges listed on the black market uplink - surely these won't explode within your own hands..."
	item = /obj/item/grenade/c4
	price_min = CARGO_CRATE_VALUE * 4
	price_max = CARGO_CRATE_VALUE * 6
	stock_max = 4
	availability_prob = 30
```

I am very very iffy on explosives in general being available through it, if they are to be they need to be a lot more expensive and rare to show up.

**Review by SomeRandomOwl — DISMISSED** [review]

**Comment by desminus** [comment]
Ideally i like the availibity of the stuff in there, though- the explosives and tram remote control, are too much for it imo.

**Comment by honkpocket** [comment]
I've decided to remove the explosives, for now.

**Comment by Samarai** [comment]
I think the prices for some of these items are a bit too high (judged purely on the image shown in this PR and me glancing at the code, so it might not be an accurate representation of the 'average' prices), at least in comparison to roughly-equivalent items sold in gun-cargo and our changed prices for the BMU, but that might be an issue with gargo power creep/its prices being too low. 

The bullet protection value of bulletproof armor is matched if not exceeded by sacrificial armor, which is cheaper than it and won't give security an excuse to pester you because anyone can buy sacrificial armor from cargo, the only downside it has being that it has limited durability. (but i have literally never a sacrificial vest/helmet take enough damage to break even in a round where i've gotten in multiple gunfights) Plus, you're guaranteed to actually get armor bought from cargo, because it isn't sent to a random, possibly inaccessible place.

The syndicate mask is definitely way overpriced for what it offers, which is just having pepperspray protection and a small item size so it can fit in boxes unlike the full-sized gas mask, but it isn't hard to deal with the larger size of that _and_ anyone can start with a small gun case for the express purpose of holding it alongside their other normal-sized items. 

So on and so forth, overall I do really like this PR, and these price issues are minor!

**Comment by honkpocket** [comment]
I'll pass over the prices again, I don't have a lot of experience with the economy.

**Comment by RatFromTheJungle** [comment]
> I'll pass over the prices again, I don't have a lot of experience with the economy.

rough numbers is like:
- 500 'small'
- 800 'medium'
- 1100+ 'expensive'


in reality, they don't have that much of a difference outside of slowing down assistants

**Review by Ghommie — COMMENTED** [review]
I'm confident at a couple of these items could be as well PR'd on tgstation, like the gravity grenade (at a higher price) and the ablative baseball bat, for not being found anywhere in the code outside a couple instances, as well as the syndicate stickers box maybe.

But I also think some of the prices here are too negligible, and certain items a big no no, like the no-slips shoes. You're basically selling antagonist gear at a price that any traitor can afford in its entirety if they bought a cash-loaded briefcase: 5000 credits for 3 TCs, which is lower than the TCs that'd be spent buying even a portion of what's being added here.

Also do keep in mind that once the market refactor (mirror PR) is merged, you'll be able to get the LTSRB or whatever it's called for 800 credits or less, which unlocks the option to get the goods delivered to your location at an lower fee than teleportation once built.

**Comment by honkpocket** [comment]
I hadn't accounted for that extra delivery possibility added in the refactor. 😅 I'll do another pricing pass.

The uplink items that are being added like no-slips are a core reason for the making of this PR.
A traitor _could_ min-max their TC and buy cash to then set up the LTSRB into buying the OPest items I'm making available here, but this requires a mindset that's very discouraged on Nova.

**Comment by Ghommie** [comment]
Notwithstanding the stance the administration has on antags, but I'm skeptic that anyone will get bwoinked for being 'smart' with how you invest your points, but how you use what you get afterwards.

**Comment by honkpocket** [comment]
Anti slip chameleons > Anti slip mod module
Anti slip mod module from 200-400 to 400-1000
Anti flash lenses from 200-400 to 400-1000
metal bat and gravgrenades to a TG PR

**Comment by honkpocket** [comment]
Got flagged with do-not-merge on TG, so the baseball bat and anti-grav grenades are back in this PR.

**Pull request closed without merging**